### PR TITLE
feat: add logout to public form header on submission end page (if auth'd), pass colorscheme to endpage

### LINKED ISSUE #4597: [3] signed in ID should show up on authed public form header on submission end page
Currently does not

Screenshot 2022-08-17 at 1 58 57 PM

### PR DESCRIPTION
**Pull request by karrui**
## Problem

This PR adds a logout button on submission end page if user is logged in, to better be in line with design.

Closes #4597.

## Solution

**Features**:

- feat: add logout button to form banner endpage

### CODE REVIEW

File: frontend/src/features/public-form/components/FormStartPage/FormBannerLogo.tsx [hunk 1]
```diff
@@ -56,15 +80,23 @@ export const FormBannerLogo = ({
   if (!hasLogo) return null
```
Comment (justynoh) [inline frontend/src/features/public-form/components/FormStartPage/FormBannerLogo.tsx]: If there's no logo then we don't actually display this form logout button at all. is that intended? seems a bit strange...

File: frontend/src/features/public-form/components/FormStartPage/FormLogoutButton.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        />
+      ) : (
+        <Button
+          colorScheme={colorScheme}
+          variant="clear"
+          rightIcon={<BiLogOutCircle fontSize="1.5rem" />}
+          onClick={onLogout}
+        >
+          {loggedInId} - Sign out
+        </Button>
```
Comment (justynoh) [inline frontend/src/features/public-form/components/FormStartPage/FormLogoutButton.tsx]: seems that we always show this as a button even if `onLogout` is not provided (specifically, we don't provide the `onLogout` prop in the `EndPageView`) 

Should we disable the button in this case? Also, it just seems a bit weird that we allow for "logging out" after the form is submitted anyway. Maybe we should just have something that says "You have been successfully logged out"...

File: frontend/src/features/public-form/components/FormStartPage/FormBannerLogo.tsx (hunk 1 above)
Comment (karrui) [reply frontend/src/features/public-form/components/FormStartPage/FormBannerLogo.tsx]: good catch. will talk to @pearlyong

**Comment by karrui** [comment]
Draft, working on new screens provided in the issue

**Comment by karrui** [comment]
new designs implemented.

File: frontend/src/features/public-form/components/FormStartPage/FormBannerLogo.tsx (hunk 1 above)
Comment (karrui) [reply frontend/src/features/public-form/components/FormStartPage/FormBannerLogo.tsx]: will always display a logo now on end page (defaults to form logo, follows pearly's new designs)

File: frontend/src/features/public-form/components/FormStartPage/FormLogoutButton.tsx (hunk 2 above)
Comment (karrui) [reply frontend/src/features/public-form/components/FormStartPage/FormLogoutButton.tsx]: disabled button if onLogout is not provided

>Also, it just seems a bit weird that we allow for "logging out" after the form is submitted anyway.

There used to be a "remember me" checkbox in AngularJS that kept the session alive for super long, but we have removed it. However, session still lasts for 24 hours, we should allow for users to log out regardless!

**Comment by justynoh** [comment]
no change to stories, but there seems to be a visual regression for custom logo start page D:

**Comment by karrui** [comment]
> no change to stories, but there seems to be a visual regression for custom logo start page D:

aiya, forgot update locations for the other stuff. form start page story regression is expected since form logo is now decoupled from start page. will fix and add new stories regardless

**Review by justynoh — APPROVED** [review]
lgtm

**Comment by justynoh** [comment]
my favorite story is "No Logo"

**Pull request merged**